firehose.c

```diff
@@ -53,6 +53,11 @@
 #include "qdl.h"
 #include "ufs.h"
 
+enum {
+	FIREHOSE_ACK,
```

Comment:
"Clean this up by defining ACK and NAK in the positive value space and
use the negative value space for errno."

almost, but I don't mind ;)

maybe add a =value to the first entry